packages/core/test/controls/controlsTests.tsx

```diff
@@ -31,7 +31,17 @@ describe("Controls:", () => {
         });
     });
 
-    controlsTests(Switch, "checkbox", Classes.SWITCH);
+    controlsTests(Switch, "checkbox", Classes.SWITCH, () => {
+        describe("internal text", () => {
+            const switchWithText = mount(<Switch innerLabelChecked="Checked text" innerLabel="Unchecked text" />);
+            it("renders internal text", () => {
```

Comment:
please add some more tests and we're good:

1. no text elements rendered by default
2. `innerLabel` only used for both states
3. `innerLabelChecked` used for checked state